src/containers/Downloads/DownloadDetails.js

```diff
@@ -64,7 +65,9 @@ const SpeciesSamples = ({ samplesBySpecies, removeSpecies }) => {
   return Object.keys(species).map((speciesName, i) => (
     <div className="downloads__sample" key={i}>
       <div className="downloads__sample-info">
-        <h2>{speciesName} Samples</h2>
+        <h2 className="downloads__species-title">
+          {formatSentenceCase(speciesName)} Samples
```

Comment:
ooo! Clever use of formatSentenceCase to do this! 😀

Reply:
Realized I should also do this in the two other spots the species name appears on the downloads page